comp/api/api/utils/stream/stream.go

```diff
@@ -93,3 +96,23 @@ func GetStreamFunc(messageReceiverFunc func() MessageReceiver, streamType, agent
 		}
 	}
 }
+
+// OpenFileForWriting opens a file for writing
+func OpenFileForWriting(filePath string) (*os.File, *bufio.Writer, error) {
+	log.Infof("opening file %s for writing", filePath)
```

Comment:
Would it make sense to add some context to this message, like why we are opening the file? Consider that the user will see the line in the logs out of context of the surrounding code.